app/models/miq_schedule/import_export.rb

```diff
@@ -61,7 +61,7 @@ def import_from_hash(miq_schedule, _options = nil)
       miq_schedule = handle_miq_report_attributes_for_import(miq_schedule) if miq_schedule["resource_type"] == "MiqReport"
 
       unless miq_schedule['userid'] == 'admin' || miq_schedule['userid'] == 'system'
-        miq_schedule['userid'] = User.find_by(:id => report["userid"])
+        miq_schedule['userid'] = User.find_by(:id => miq_schedule['userid'])
```

Comment:
@lpichler  why we need to replace string id with actual object if user is not admin?
Is it to make sure that ```miq_schedule['userid'] ``` set to nil if user not found ?

Reply:
@lpichler sorry to nag but I need this to get answered before I can get this merged

Reply:
I think it could be merged anyway, since it does not work as is...

Reply:
yes it looks useless, to I changed it here  https://github.com/ManageIQ/manageiq/pull/19322

Reply:
> Is it to make sure that `miq_schedule['userid'] ` set to nil if user not found ?

yes, it looks like origin purpose